apis/core/v1alpha2/feature_stability_policies.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha2
+
+// Policy represents Stability level policy
+type Policy struct {
```

Comment:
nit:  could we add some inline comments that briefly describe what the following four policies mean? I guess some of them are not self-explanation, e.g, `VoidWarranty`. And I guess not all people have read the design doc.

Reply:
i have a minor cleanup commit coming, i already added these changes to that commit